src/main/java/appeng/integration/modules/NEIHelpers/NEICraftingHandler.java

```diff
@@ -44,8 +44,8 @@
 public class NEICraftingHandler implements IOverlayHandler
 {
 
-	final int offsetX;
-	final int offsetY;
+	private final int offsetX;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "offsetX" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)